app/src/main/java/com/example/countryinfo/Apollo.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.countryinfo
+
+import android.content.Context
+import android.os.Looper
+import com.apollographql.apollo.ApolloClient
+
+private var instance: ApolloClient? = null
+
+fun apolloClient(context: Context): ApolloClient {
+    check(Looper.myLooper() == Looper.getMainLooper()) {
+        "Only the main thread can get the apolloClient instance"
+    }
+
+    if (instance != null) {
+        return instance!!
+    }
+
+    instance = ApolloClient.builder()
+        .serverUrl("https://countries-274616.ew.r.appspot.com/")
```

Comment:
Let's move it to constant. Companion object would be fine for now.

Reply:
Done